index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section class="Mobile_background">
+
+    
+        <header>
+        <img src="images/My_Logo.png" class="logo" alt="Logo">
+        <img src="images/menu.png" class="menu" alt="Menu">
+        </header>
+       <!-- Headline section-->
+        <div class="headline_section">
+            <img src="images/Ellipse_11.svg" class="ellipse_11" alt="ellipse_11"> 
+            <img src="Images/Rectangle_6.svg" class="rectangle_6" alt="ellipse_6">
+            <img src="Images/Ellipse_5.svg" class="ellipse_5" alt="ellipse_5">
+            <img src="Images/Rectangle_7.svg" class="rectangle_7" alt="rectangle_7">
+            <img src="Images/Rectangle_4.svg" class="rectangle_4" alt="rectangle_4">
+            <img src="images/Ellipse_3.svg" class="ellipse_3" alt="ellipse_3">
+            <h1>I’m Mounir <br>
+                Glad to see <br>you! </h1>
+          
+            <div class="p1">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+                 and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+          
+            <div class="p2">
+                 <p>LET’S CONNECT</p>
+                </div>
```

Comment:
- [ ] [Optional] I strongly suggest you delete the `div` that is wrapping the `p` tag. From my point of view is meaningless to do that, you could directly apply any style you require in the `p` tag